docs/asciidoc/static/include/pluginbody.asciidoc

```diff
@@ -439,7 +439,7 @@ endif::encode_method[]
 ==== Configuration Parameters
 [source,ruby]
 ----------------------------------
-  config :variable_name, :validate => :variable_type, :default => "Default value", :required => boolean, :deprecated => boolean
+  config :variable_name, :validate => :variable_type, :default => "Default value", :required => boolean, :deprecated => boolean, :obsolete => string
```

Comment:
Indeed. Also, I noticed that the docs for `:deprecated` are incorrect. It's not a boolean (although it can be), it's a String. I opted not to fix that in this PR to avoid confusing things, oops ;P